### ISSUE TITLE
Support for @DecimalMax

### ISSUE THREAD

**Issue by aleks-p** [netceteragroup/valdr#88]
Currently, the way to do a max-validation against a decimal limit is via a custom validator.

Can you consider adding support for the DecimalMax constraint (part of the built-in constraints of Bean validation)?

**Comment by marcelstoer** [comment]
You are right, there are a few Bean Validation constraints that are not supported yet. As per https://github.com/netceteragroup/valdr-bean-validation#mapping-of-bean-validation-constraints-to-valdr-constraints and http://beanvalidation.org/1.1/spec/#builtinconstraints (inverse intersection) they are:
- DecimalMin/Max
- AssertTrue/False
- Null

We will consider adding support for them in an upcoming release.
